packages/cli/lib/prompts/createProjectPrompt.js

```diff
@@ -1,23 +1,45 @@
 const path = require('path');
 const { getCwd } = require('@hubspot/cli-lib/path');
-const { PROJECT_COMPONENT_TYPES } = require('@hubspot/cli-lib/lib/constants');
+const {
+  PROJECT_COMPONENT_TYPES,
+  PROJECT_PROPERTIES,
+} = require('@hubspot/cli-lib/lib/constants');
 const { promptUser } = require('./promptUtils');
 const { fetchJsonFromRepository } = require('@hubspot/cli-lib/github');
 const { i18n } = require('@hubspot/cli-lib/lib/lang');
+const { logger } = require('@hubspot/cli-lib/logger');
 
 const i18nKey = 'cli.lib.prompts.createProjectPrompt';
 
-const createTemplateOptions = async () => {
+const hasAllProperties = projectList => {
+  return projectList.every(config =>
+    PROJECT_PROPERTIES.every(p =>
+      Object.prototype.hasOwnProperty.call(config, p)
+    )
+  );
+};
+
+const createTemplateOptions = async repoPath => {
+  const isRepoPath = !!repoPath;
   const config = await fetchJsonFromRepository(
-    'hubspot-project-components',
-    'main/config.json'
+    repoPath,
+    'main/config.json',
+    isRepoPath
   );
 
+  if (!config[PROJECT_COMPONENT_TYPES.PROJECTS]) {
```

Comment:
We should probably add `process.exit` calls to the checks here. I also ran into an error in my console (screenshot above) where `config` is undefined. It may be worth adding a "not null" check at the start of this if.

Reply:
Fixed in https://github.com/HubSpot/hubspot-cli/pull/837/commits/3d549ea4cb140ba9d558e364c834c2dd7aa11a9c